data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb

```diff
@@ -0,0 +1,686 @@
+    "\n",
+    "We may want to rethink the denominator of our equation for housing cost burden.\n",
+    "\n",
+    "\"Right now it’s all housing units with a cost burden computed. \n",
+    "\n",
+    "Alternatively, you could use low-income households (with cost burden computed) as the denominator, which would be a measure of relative cost burden just for low-income households. \n",
+    "\n",
+    "Both approaches are appropriate, but they tell a different story. You can imagine an area with few low-income households but a vast majority of them being cost burdened. In your calculation, you’d get a small percentage. \n",
+    "\n",
+    "In the alternative approach, it’s a large percentage. Just something to think about. It depends on the story you want to tell.\"\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import math\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Table of Contents"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Comparisons\n",
+    "*  [Implementation by Lucas](#lucas)\n",
+    "*  [Implementation by Saran](#saran)\n",
+    "*  [Side-by-side Comparison](#comparison)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The relevant denominator variables - all with line type of subtotal - in table 8 of the CHAS dataset are the following (CHAS data dictionary available [here](https://www.huduser.gov/portal/datasets/cp/CHAS-data-dictionary-14-18.xlsx)):\n",
+    "\n",
+    "|   Name  |                          Label                      |\n",
+    "|---------|-----------------------------------------------------|\n",
+    "|T1_est3  |      Owner occupied less or equal to 30% of HAMFI   |                                                    | \n",
+    "|T8_est16 |            Owner occupied greater than 30% of HAMFI cost burden less than 50%           |\n",
+    "|T8_est29 |Owner occupied greater than 50% but less than or equal to 80% of HAMFI\tcost burden |\n",
+    "|T8_est69 |Renter occupied less than or equal to 30% of HAMFI|\n",
+    "|T8_est82 |           Renter occupied less than or equal to 30% of HAMFI cost burden greater than 50%           |\n",
+    "|T8_est95 |Renter occupied\tgreater than 50% but less than or equal to 80% of HAMFI|\n",
+    " "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Implementation by Lucas <a id='lucas'></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read in the data from https://www.huduser.gov/portal/datasets/cp.html\n",
+    "housing = pd.read_csv(\"Table8.csv\", \n",
```

Comment:
can you make sure your code works for anyone who wants to run it? 

in this case, just copy from the ETL script to download this table and unzip it. thanks!

Reply:
Got it, thanks

Reply:
@lucasmbrown-usds I have added the ETL as per your request